source/reference/minio-mc/mc-share-list.rst

```diff
@@ -9,17 +9,20 @@
    :depth: 2
 
 .. mc:: mc share list
+.. mc:: mc share ls
 
 Syntax
 -----------
 
 .. start-mc-share-list-desc
 
-The :mc:`mc share list` command displays any unexpired presigned URLs generated
+The :mc:`mc share ls` command displays any unexpired presigned URLs generated
 by :mc:`mc share upload` or :mc:`mc share download`
 
 .. end-mc-share-list-desc
 
+The :mc:`mc share list` alias has equivalent functionality to :mc:`mc share ls`.
```

Comment:
`alias` swap for `command` since `alias` has a specific meaning for us in the docs.